lumicks/pylake/nb_widgets/fd_selector.py

```diff
@@ -0,0 +1,136 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import time
+
+
+class FdRangeSelectorWidget:
+    def __init__(self, fd_curve, ax=None, show=True):
+        self._ax = ax if ax else plt.axes(label=f"lk_widget_{time.time()}")
```

Comment:
Is `_ax` supposed to mean axes? If so maybe we should consider just renaming it to `_axes`

Reply:
Yes. 👍
Done!